config/rukpak/apis/webhooks/resources/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: webhooks
           command: ["/webhooks"]
-          image: quay.io/operator-framework/rukpak:v0.8.0
+          image: quay.io/operator-framework/rukpak:main
```

Comment:
Are using main as a temporary measure? I mainly ask as this image may break in combination with the static manifests we're using here over time.

Reply:
Also, we may be able to turn a kustomize dial that allows us to set this image everywhere with one line making it easier to change overtime.

Reply:
Yeah the better change here is using whatever kustomize can offer here. That would be a good follow-up change.

Reply:
https://github.com/openshift/platform-operators/issues/21